docs/docsite/rst/guide_migration.rst

```diff
@@ -18,6 +18,9 @@ Migrated Modules
 The following modules have been migrated in some shape or form into this collection
 
 * ``community.windows.win_iis_website`` -> ``microsoft.iis.website`` - :ref:`details <ansible_collections.microsoft.iis.docsite.guide_migration.migrated_modules.win_iis_website>`
+* ``community.windows.win_iis_virtualdirectory`` -> ``microsoft.iis.virtual_directory`` - :ref:`details <ansible_collections.microsoft.iis.docsite.guide_migration.migrated_modules.win_iis_virtualdirectory>`
```

Comment:
We should add an entry for `win_iis_webbinding` and a section for that module that shows how to use `microsoft.iis.website` instead.